third_party/blink/renderer/modules/mediastream/webmediaplayer_ms_compositor.cc

```diff
@@ -119,14 +121,33 @@ scoped_refptr<media::VideoFrame> CopyFrame(
                        new_frame->data(media::VideoFrame::kVPlane),
                        new_frame->stride(media::VideoFrame::kVPlane),
                        coded_size.width(), coded_size.height());
+      if (frame->format() == media::PIXEL_FORMAT_I420A) {
+        libyuv::CopyPlane(frame->data(media::VideoFrame::kAPlane),
+                          frame->stride(media::VideoFrame::kAPlane),
+                          new_frame->data(media::VideoFrame::kAPlane),
+                          new_frame->stride(media::VideoFrame::kAPlane),
+                          coded_size.width(), coded_size.height());
+      }
+    } else if (frame->format() == media::PIXEL_FORMAT_I444) {
+      libyuv::CopyPlane(frame->data(media::VideoFrame::kYPlane),
```

Comment:
can we use libyuv::I444Copy in here?
I see that above is used libyuv::I420Copy and libyuv::NV12Copy, but here we use libyuv::CopyPlane call 3 times, so I was wondering whether those three call can be replaced with one libyuv::I444Copy?

Reply:
Last time we checked there was no such function and we wanted to avoid committing code to yet another project. But yes, the cleanest way to do this would be to add an I444Copy function to libyuv and use it here.